lib/route/ROUTING.md

```diff
@@ -0,0 +1,446 @@
+# Obelisk Routing
+
+The `obelisk-routes` package is designed to help with managing the paths and parameters for
+routing in your application.
+
+This is distinct from packages like `servant` that are designed for specifying endpoints for a
+REST API. When building routes with `obelisk-routes` you're not going to be handling headers or
+HTTP methods.
+
+One of the goals of `obelisk-routes` is to allow for complex routing structures to be partitioned
+into manageable chunks that are then combined in a straight-forward manner. Allowing for re-use
+of various components and to help the type-system provide as many guarantees as possible. We'll
```

Comment:
"Allowing for ..." isn't a full sentence. Maybe "This allows for..."

Reply:
resolved by deletion as part of previous comment